Formula/gimme-aws-creds.rb

```diff
@@ -0,0 +1,75 @@
+class GimmeAwsCreds < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI to retrieve AWS credentials from Okta"
+  homepage "https://github.com/Nike-Inc/gimme-aws-creds"
+  url "https://github.com/Nike-Inc/gimme-aws-creds/archive/v2.0.2.tar.gz"
+  sha256 "32a28f4d8fc6a72a3a53ab822030f0d00671a79fd4d86823e16464482ee7c664"
+
+  depends_on "python"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system libexec/"bin/pip", "install", "-v", "--no-binary", ":all:",
+                              "--ignore-installed", buildpath
+    system libexec/"bin/pip", "uninstall", "-y", "gimme-aws-creds"
+    venv.pip_install_and_link buildpath
+  end
+
+  test do
+    require "pty"
+
+    touch ".okta_aws_login_config"
+
+    PTY.spawn("#{bin}/gimme-aws-creds --configure") do |input, output, _pid|
```

Comment:
Can you describe at a high-level what this is doing and I'll attempt to help make this test a little nicer, thanks!

Reply:
Similar to the test for `aws-okta.rb` this is configuring app's settings. First run the application asks for a number of details and this is checking that this flow works, entering only required fields and accepting defaults.

Reply:
Can this be entered with a pipe e.g. `echo foo | gimme-aws-creds`?

Reply:
@Firefishy nudge on this; cleaning up the test is the last thing I'd like to do before merging this.

Reply:
@MikeMcQuaid yes, I was looking into switching it to a `echo -e` pipe, but not been able to get it to work yet.

Reply:
The `pipe_output` helper would be our preferred method of doing this if possible. I'm happy to help with that if I can! Let me know.

Reply:
I've now switched to using `pipe_output` for running the configure test.